src/commands/database/mysql/import.tsx

```diff
@@ -0,0 +1,132 @@
+
+export class Import extends ExecRenderBaseCommand<
+  typeof Import,
+  Record<string, never>
+> {
+  static summary = "Imports a dump of a MySQL database";
+  static flags = {
+    ...processFlags,
+    ...mysqlConnectionFlagsWithTempUser,
+    ...sshConnectionFlags,
+    input: Flags.string({
+      char: "i",
+      summary: 'the input file from which to read the dump ("-" for stdin)',
+      description:
+        'The input file from which to read the dump to. You can specify "-" or "/dev/stdin" to read the dump directly from STDIN.',
+      required: true,
+    }),
+    gzip: Flags.boolean({
+      summary: "uncompress the dump with gzip",
+      aliases: ["gz"],
+      description:
+        "Uncompress the dump with gzip while importing. This is useful for large databases, as it can significantly reduce the size of the dump.",
+      default: false,
+      required: false,
+    }),
+  };
+  static args = { ...mysqlArgs };
+
+  protected async exec(): Promise<Record<string, never>> {
+    const databaseId = await withMySQLId(this.apiClient, this.flags, this.args);
+    const p = makeProcessRenderer(this.flags, "Importing a MySQL database");
+
+    const connectionDetails =
+      await getConnectionDetailsWithPasswordOrTemporaryUser(
+        this.apiClient,
+        databaseId,
+        p,
+        this.flags,
+      );
+
+    const { project } = connectionDetails;
+    const mysqlArgs = buildMySqlArgs(connectionDetails);
+
+    let cmd: RunCommand = { command: "mysql", args: mysqlArgs };
+    if (this.flags.gzip) {
+      const escapedArgs = shellEscape(mysqlArgs);
+      cmd = {
+        shell: `set -e -o pipefail > /dev/null ; gunzip | mysqldump ${escapedArgs}`,
+      };
+    }
+
+    await p.runStep(
+      <Text>
+        starting mysql via SSH on project <Value>{project.shortId}</Value>
+      </Text>,
+      () =>
+        executeViaSSH(
+          this.apiClient,
+          this.flags["ssh-user"],
+          { projectId: connectionDetails.project.id },
+          cmd,
+          null,
+          this.getInputStream(),
+        ),
+    );
+
+    await p.complete(
+      <ImportSuccess
+        database={connectionDetails.database}
+        input={this.flags.input}
+      />,
+    );
+
+    return {};
+  }
+
+  protected render(): ReactNode {
+    return undefined;
+  }
+
+  private getInputStream(): NodeJS.ReadableStream {
+    if (this.flags.input === "-") {
+      return process.stdin;
+    }
+
+    return fs.createReadStream(this.flags.input);
+  }
+}
+
+function ImportSuccess({
+  database,
+  input,
+}: {
+  database: string;
+  input: string;
+}) {
+  return (
+    <Success>
+      Dump of MySQL database <Value>{database}</Value> successfully imported{" "}
+      from <Value>{input}</Value>
+    </Success>
+  );
+}
+
+function buildMySqlArgs(d: {
+  hostname: string;
+  user: string;
+  password: string;
+  database: string;
+}): string[] {
+  return ["-h", d.hostname, "-u", d.user, "-p" + d.password, d.database];
```

Comment:
```suggestion
  return ["-h", d.hostname, "-u", d.user, `-p${d.password}`, d.database];
```

in my opinion, this would make it more optional, that this is on purpose and not a typo (`+` instead of `,`)

Reply:
Fixed.